doc/usage.rst

```diff
@@ -50,6 +48,13 @@ show [OPTION…] *PACKAGE*
 
    -s, --size              Print the size of each file
 
+validate [OPTION…] *PACKAGE* *SIZE*
+   Validate a partup PACKAGE
+
+   -s, --skip-checksums         Skip checksum verification for all input files
+   -w, --write-file             Install the package to a file as an additional validation step
+   -k, --keep-file-as=FILENAME  Keep the file. Implies --write-file
```

Comment:
From a user perspective I think it is not clear, what you mean exactly with "file" in the options `--write-file` and `--keep-file-as` as the term collides with the input files.